conda_build_all/resolved_distribution.py

```diff
@@ -1,47 +1,13 @@
 from __future__ import print_function
 
-from contextlib import contextmanager
-import logging
-import os
-import subprocess
-from argparse import Namespace
-
 from conda.api import get_index
-from conda_build.metadata import MetaData
 import conda.config
 import conda_build.config
 
 
 #import conda_build_all.logging
 import conda_build_all.version_matrix as vn_matrix
-
-
-@contextmanager
-def setup_vn_mtx_case(case):
-    orig_npy = conda_build.config.config.CONDA_NPY
-    orig_py = conda_build.config.config.CONDA_PY
-    orig_r = conda_build.config.config.CONDA_R
-    orig_perl = conda_build.config.config.CONDA_PERL
-
-    for pkg, version in case:
-        if pkg == 'python':
-            version = int(version.replace('.', ''))
-            conda_build.config.config.CONDA_PY = version
-        elif pkg == 'numpy':
-            version = int(version.replace('.', ''))
-            conda_build.config.config.CONDA_NPY = version
-        elif pkg == 'perl':
-            conda_build.config.config.CONDA_PERL = version
-        elif pkg == 'r':
-            conda_build.config.config.CONDA_R = version
-        else:
-            raise NotImplementedError('Package {} not yet implemented.'
-                                      ''.format(pkg))
-    yield
-    conda_build.config.config.CONDA_NPY = orig_npy
-    conda_build.config.config.CONDA_PY = orig_py
-    conda_build.config.config.CONDA_R = orig_r
-    conda_build.config.config.CONDA_PERL = orig_perl
+from conda_build_all.version_matrix import setup_vn_mtx_case
 
 
 class ResolvedDistribution(object):
```

Comment:
Starting to wonder how much value there is in this class with config being attached to the MetaData class now.